lib/packwerk/cli.rb

```diff
@@ -154,11 +154,15 @@ def validate(_paths)
     end
 
     sig { returns(ApplicationValidator) }
-    def checker
-      Packwerk::ApplicationValidator.new(
-        config_file_path: @configuration.config_path,
-        configuration: @configuration,
-        environment: @environment,
+    def validator
+      Packwerk::ApplicationValidator.new
+    end
+
+    sig { returns(PackageSet) }
+    def package_set
```

Comment:
I had to define the package set on the run_context in parse_run to cleanup todos. I wonder if we DRY things up somehow.